config/sync/core.extension.yml

```diff
@@ -41,12 +42,14 @@ module:
   system: 0
   taxonomy: 0
   text: 0
+  token: 0
```

Comment:
I didn't see token as a requirement in the composer file, has it been included?

Reply:
Token and ctools are installed as dependencies of pathauto. I haven't ever noticed that they have appeared in composer before when installed as dependencies in this way, should they be?

Reply:
Ah that's grand. I'd just be careful that if you go on to use the token or ctools outside the context of pathauto that you include those in composer.